src/server/schema.js

```diff
@@ -146,12 +146,20 @@ const getAggregationType = (entry) => {
   return '';
 };
 
-const getAggregationSchemaForOneIndex = (esInstance, esIndex, esType) => {
+const getAggregationSchemaForOneIndex = (esInstance, esDict) => {
+  const esIndex = esDict.index;
+  const esType = esDict.type;
+  let histogramTypePrefix = '';
+  // eslint-disable-next-line no-console
+  console.log('>>>>> inside getAggregationSchemaForOneIndex; esIndex: ', esDict);
+  if (Object.prototype.hasOwnProperty.call(esDict, 'tier_access_level') && esDict.tier_access_level === 'regular') {
+    histogramTypePrefix = 'RegularAccess';
```

Comment:
better put this value as a const and use ternary operator to decide whether to include it in a string or not, rather than define it using let and calling repeatly, there are 3-4 more places down below has the similar structure